packages/schematics/angular/application/index.ts

```diff
@@ -50,7 +50,7 @@ function addDependenciesToPackageJson(options: ApplicationOptions) {
       {
         type: NodeDependencyType.Dev,
         name: 'typescript',
-        version: latestVersions.TypeScript,
+        version: latestVersions.typescript,
```

Comment:
What do you think about using indexed property access for all the package.json originating properties?